chia/wallet/nft_wallet/nft_wallet.py

```diff
@@ -530,9 +528,41 @@ async def transfer_nft(
             name=bytes32(token_bytes()),
             memos=list(compute_memos(full_spend).items()),
         )
-        await self.standard_wallet.push_transaction(nft_record)
-        full_spend.debug()
-        return full_spend
+        return nft_record
+
+    async def update_metadata(self, nft_coin_info: NFTCoinInfo, uri: str) -> Optional[SpendBundle]:
+        coin = nft_coin_info.coin
```

Comment:
Optional comment, may be distinguish  this with NFT1 approach. Like add a if statement or change the method name to update_metadata_nft0

Reply:
ok, makes sense, will add support this in a separate PR